### PR TITLE
Change icons for map search

### PR DESCRIPTION
**Pull request by woeltjen**
## Description
Follow up on discussion from #289: Improve icons for hotels and destinations

## How to Test
1. Search for some hotels
2. Enter a "Close To..." location
3. Do the markers on the map make sense?

Which devices did you test on?
- [ ] Chrome on Mac
- [ ] Chrome on PC
- [ ] Firefox on Mac
- [ ] Firefox on PC
- [ ] Safari iPhone
- [ ] Chrome Android

## REVIEWERS:
Check against these principles:

### High level
Does this code need to be written?
What are the alternatives?
Will this implementation become a support issue?
How much error margin does this solution have?

### Code
* Does the code follow industry standards?
JS: https://github.com/airbnb/javascript
React: https://github.com/airbnb/javascript/tree/master/react
https://github.com/vasanthk/react-bits
Documentation headers: http://usejsdoc.org/index.html

* Is there duplicated code? Can it be refactored into a shared method?
* Is the code consistent with our project?
* Are there unit tests? Do they test the states?
* Is the person refactoring another developer's code? If possible, did the original developer approve?

### Variables/Naming:
* Would the variable type led to future edge cases?
* Are the variable naming clear? Would the value contain something other than what the name describes.

### Security
* Can this be hacked or abused by the user?

### CODE REVIEW

**Comment by woeltjen** [comment]
image

Fond of the ★ marker; I recall that in the UI prototype we had some trouble distinguishing hotels from the other destinations, trying out the combination of home-vs-star to see if that helps.

**Comment by tc** [comment]
star as destination works. 
home as home/hotel works.

where's the driving path from DG to coit tower in the map?

**Comment by kcvan** [comment]
> where's the driving path from DG to coit tower in the map?

*running path😂

**Comment by kcvan** [comment]
looks great, vic!

File: src/legacy/shared/GoogleMapsWithMarkers/GoogleMapsWithMarkers.tsx [hunk 1]
```diff
@@ -11,12 +11,16 @@ import { LatLngBounds, ListingShort } from 'networking/listings';
 
 import GoogleMapsWithMarkersContainer from './GoogleMapsWithMarkers.container';
 
+const hotelMarker = require('assets/images/iconmonstr-location-12-32.png');
```
Comment (kcvan) [inline src/legacy/shared/GoogleMapsWithMarkers/GoogleMapsWithMarkers.tsx]: i think we can upload these to s3 and use it there

Comment (tc) [reply src/legacy/shared/GoogleMapsWithMarkers/GoogleMapsWithMarkers.tsx]: either way, the icons are still cached via cloudfront.

**Pull request merged**